javaparser-core/src/main/java/com/github/javaparser/ast/nodeTypes/NodeWithMembers.java

```diff
@@ -112,6 +113,21 @@ default FieldDeclaration addField(Type type, String name, Modifier... modifiers)
     }
 
     /**
+     * Add a field to this
+     *
+     * @param type the type of the field
+     * @param name the name of the field
+     * @param initializer the initializer of the field
+     * @param modifiers the modifiers like {@link Modifier#PUBLIC}
+     * @return the {@link FieldDeclaration} created
+     */
+    default FieldDeclaration addField(Type type, String name, Expression initializer, Modifier... modifiers) {

```

Comment:
How about calling it `addFieldWithInitializer` ?

Reply:
To me either name is perfect. Should I add it?

Reply:
Yes please, I think it would make things clearer